components/molecules/dropdown/components/colourDropdown.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Organisms - Color picker
+ */
+
+// React
+import React from 'react'
+import { string, func } from 'prop-types'
+import { Position } from 'components/theme'
```

Comment:
@mhsnprvr2 Not sure the aliases will work here outside of IUI - for instance when I import into the portal - be careful with the paths here

Reply:
@mhsnprvr2 check this file no comments are resolved